config.h

```diff
@@ -8,7 +8,10 @@
 
 
 /* Repository URL */
-#define OSLIST_URL                        "https://raw.githubusercontent.com/analogdevicesinc/adi-kuiper-imager/update_imager/os_list.json"
+#define OSLIST_URL                        "https://raw.githubusercontent.com/analogdevicesinc/adi-kuiper-imager/qml/os_list.json"
+
```

Comment:
Should these links be valid? It looks like they are not... Or will we add them in the future?

Reply:
will become valid once we merge this.